app/controllers/tasks_controller.rb

```diff
@@ -85,7 +85,7 @@ def add_to_collection
   end
 
   def download
-    zip_file = ProformaService::ExportTask.call(task: @task)
+    zip_file = ProformaService::ExportTask.call(task: @task, options: {version: params[:version]})
```

Comment:
When manipulating the URL (in my browser) and specifying an unsupported version (such as `1.0`) an unhandled exception is thrown (`ProformaXML::PostGenerateValidationError "version not supported"`). Maybe we could handle this error, so that we don't respond with an internal server error in production?